bedrock/settings/base.py

```diff
@@ -406,6 +406,19 @@ def lazy_langs():
 SESSION_COOKIE_SECURE = not DEBUG
 SESSION_ENGINE = 'django.contrib.sessions.backends.cache'
 
+# legacy setting. backward compat.
+DISABLE_SSL = config('DISABLE_SSL', default=True, cast=bool)
```

Comment:
@jgmize so what do you think. Is it worth me adding the new config to every bedrock deis app now and removing this line before we merge, or should we do that later? I'm on the fence. It seems a bit disruptive, but it would be nice for this to be cleaner.